packages/desktop-client/src/components/transactions/TransactionsTable.js

```diff
@@ -838,7 +843,7 @@ const Transaction = memo(function Transaction(props) {
         isPreview && { color: colors.n5, fontStyle: 'italic' },
         _unmatched && { opacity: 0.5 },
       ]}
-      onMouseEnter={() => onHover && onHover(transaction.id)}
+      onPointerEnter={() => onHover && onHover(transaction.id)}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why change this? Genuinely interested if one of these is better than the other.

Reply:
Based on my understanding, pointer events are just like mouse events but with better support for touch and pen devices.